### PR TITLE
Bump policy-controller to v0.4.0

### PR DESCRIPTION
**Pull request by hectorj2f**
Signed-off-by: Hector Fernandez <hector@chainguard.dev>

## Description of the change

Bump the policy-controller chart to use v0.4.0

## Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). Where applicable, update and bump the versions in any associated umbrella chart
- [x] Variables are documented in the `values.yaml` and added to the README.md. The [helm-docs](https://github.com/norwoodj/helm-docs) utility can be used to generate the necessary content. Use `helm-docs --dry-run` to preview the content.
- [x] JSON Schema generated.
- [x] List tests pass for Chart using the [Chart Testing](https://github.com/helm/chart-testing) tool and the `ct lint` command.

### CODE REVIEW

File: charts/policy-controller/templates/policy-webhook/configmap-policy-controller.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  name: config-policy-controller
+  namespace: {{ .Release.Namespace }}
+data:
+  _example: |
+    ################################
+    #                              #
+    #    EXAMPLE CONFIGURATION     #
+    #                              #
+    ################################
+    no-match-policy: warn
```
Comment (cpanato) [inline charts/policy-controller/templates/policy-webhook/configmap-policy-controller.yaml:27]: nit: new line

File: charts/policy-controller/README.md [hunk 2]
```diff
@@ -24,14 +24,13 @@ The Helm chart for Policy  Controller
...
 | cosign.cosignPub | string | `""` |  |
-| cosign.secretKeyRef.name | string | `""` |  |
 | cosign.webhookName | string | `"policy.sigstore.dev"` |  |
 | imagePullSecrets | list | `[]` |  |
 | policywebhook.env | object | `{}` |  |
 | policywebhook.extraArgs | object | `{}` |  |
 | policywebhook.image.pullPolicy | string | `"IfNotPresent"` |  |
 | policywebhook.image.repository | string | `"ghcr.io/sigstore/policy-controller/policy-webhook"` |  |
-| policywebhook.image.version | string | `"sha256:d1e7af59381793687db4673277005276eb73a06cf555503138dd18eaa1ca47d6"` |  |
+| policywebhook.image.version | string | `"sha256:03f6b9807103c988439741fdc2ec4410a85c13ba62fbad58448a070ac07bb5bc"` |  |
```
Comment (cpanato) [inline charts/policy-controller/README.md:33]: can you add a version in the end as well?

**Review by cpanato — COMMENTED** [review]
thanks for doing this

File: charts/policy-controller/README.md (hunk 2 above)
Comment (hectorj2f) [reply charts/policy-controller/README.md]: done

File: charts/policy-controller/templates/policy-webhook/configmap-policy-controller.yaml (hunk 1 above)
Comment (hectorj2f) [reply charts/policy-controller/templates/policy-webhook/configmap-policy-controller.yaml]: done

**Pull request merged**